commands/user.go

```diff
@@ -387,16 +387,18 @@ func userCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	locale, _ := cmd.Flags().GetString("locale")
 	systemAdmin, _ := cmd.Flags().GetBool("system_admin")
 	emailVerified, _ := cmd.Flags().GetBool("email_verified")
+	disableWelcomeEmail, _ := cmd.Flags().GetBool("disable_welcome_email")
 
 	user := &model.User{
-		Username:      username,
-		Email:         email,
-		Password:      password,
-		Nickname:      nickname,
-		FirstName:     firstname,
-		LastName:      lastname,
-		Locale:        locale,
-		EmailVerified: emailVerified,
+		Username:            username,
+		Email:               email,
+		Password:            password,
+		Nickname:            nickname,
+		FirstName:           firstname,
+		LastName:            lastname,
+		Locale:              locale,
+		EmailVerified:       emailVerified,
+		DisableWelcomeEmail: disableWelcomeEmail,
```

Comment:
You need to add something like:

```go
UserCreateCmd.Flags().Bool("email_verified", false, "Optional. If supplied, the new user will have the email verified. Defaults to false")
```

You can see how it's done in line 243. That would register then ew flag so you can pass that in configuration. Right now would be always false, the default boolean value.

Also please add some tests to that

Reply:
In addition, CI is failing due to lack of docs. @AGMETEOR you should run `make docs` after a command change. Also we should start using dashes instead of underscores. Not sure if it would break consistency here but 1/5 on using dashes.

Reply:
@isacikgoz updated to use dashes. ```make docs``` fails atm because user Model from the server is currently not having the ```DisableWelcomeEmail``` field yet.

Reply:
@ethervoid since the ```disable-welcome-email``` has a default a unit test seems redundant. For an e2e test, it's a bit difficult to know if we actually called the ```sendWelcomeEmail``` method on the ``` EmailService``` in the server

Reply:
@AGMETEOR I was wondering if mmctl has a way to know of the notification email was sent in the e2e tests, if not we can avoid creating a test for this

Reply:
I looked through the code and dont think there's a way to do so currently. Unless @isacikgoz knows otherwise?

Reply:
Hey @isacikgoz , any final comments about this?

Reply:
`mmctl` itself is not capable of that. But in e2e tests you can use inbucket [rest api](https://pkg.go.dev/github.com/inbucket/inbucket/pkg/rest/client) to check if the email is sent or not.

Reply:
@ethervoid I am attempting to add ability to test email stuff to the ```TestHelper``` here https://github.com/mattermost/mattermost-server/pull/17072. Maybe we can easily write those tests when it successfully merges?

Reply:
Ok ,works for me